CHANGES.md

```diff
@@ -1,5 +1,11 @@
 ## [HEAD]
 
+#### New features
```

Comment:
What do you think about labelling it as something like:

- **Feature:** Accept objects in `history.push` and `history.replace` ([#141])
- **Deprecation:** Deprecate `history.pushState` and `history.replaceState` in favor of passing objects to `history.push` and `history.replace` ([#168])
- **Bugfix**: Disable browser history on Chrome iOS ([#146])
- **Bugfix**: Do not convert same-path `PUSH` to `REPLACE` if the hash has changed ([#167])

I just want an easy way for people looking at this to tell at a glance what has changed in which way - especially new features and deprecations, which might require some action on their part.